eng/Versions.props

```diff
@@ -16,13 +16,13 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Latest shared runtime version updated by darc -->
-    <MicrosoftNETCorePlatformsVersion>10.0.0-preview.5.25229.109</MicrosoftNETCorePlatformsVersion>
-    <MicrosoftNETCoreAppRefVersion>10.0.0-preview.5.25229.109</MicrosoftNETCoreAppRefVersion>
+    <MicrosoftNETCorePlatformsVersion>10.0.0-preview.6.25272.109</MicrosoftNETCorePlatformsVersion>
+    <MicrosoftNETCoreAppRefVersion>10.0.0-preview.5.25266.103</MicrosoftNETCoreAppRefVersion>
```

Comment:
@akoeplinger @ViktorHofer the rootcause seems to come from this. 103 is not a netcoreapp ref version. the xml has the right one. Any idea what happened here?

Reply:
This was a regression from https://github.com/dotnet/diagnostics/pull/5486

Reply:
@mmitche - I assume this is what was intended 
platform -> non-stable
ref -> stable version

For my learning - why is arch-specific asset not recommended for this?

Reply:
https://github.com/dotnet/diagnostics/pull/5486 missed adding the entry for Microsoft.NETCore.App.Ref to Version.Details.xml so the version wasn't getting updated and you got a mix of versions.

> For my learning - why is arch-specific asset not recommended for this?

The problem is when we build e.g. the win-x86 vertical then the Microsoft.NETCore.App.Runtime.win-x64 package isn't built so MicrosoftNETCoreAppRuntimewinx64Version would not be defined. Right now we have to manually inject properties like these with the correct value and we'd like to not have to do that.